identity-mdoc/src/main/java/com/android/identity/mdoc/connectionmethod/ConnectionMethodBle.kt

```diff
@@ -150,5 +150,21 @@ class ConnectionMethodBle(
                 map.getOrNull(OPTION_KEY_PERIPHERAL_SERVER_MODE_BLE_DEVICE_ADDRESS)?.asBstr
             return cm
         }
+
+        private fun uuidToBytes(uuid: UUID): ByteArray {
+            val data = ByteBuffer.allocate(16)
+            data.order(ByteOrder.BIG_ENDIAN)
+            data.putLong(uuid.mostSignificantBits)
+            data.putLong(uuid.leastSignificantBits)
+            return data.array()
```

Comment:
I would rewrite this to be about what's happening to ByteBuffer like so
= ByteBuffer.allocate(16).run{
   order(..)
   putLong(..)
   putLong(..)
   array()
}